cirras-underwriting-liquibase/scripts/03_00_xx/03_06_00/00/ddl/tables/cuws.create.verified_yield_contract_commodity.sql

```diff
@@ -0,0 +1,77 @@
+CREATE TABLE cuws.verified_yield_contract_commodity(
+    verified_yield_contract_commodity_guid    varchar(32)       NOT NULL,
+    verified_yield_contract_guid              varchar(32)       NOT NULL,
+    crop_commodity_id                         numeric(9, 0)     NOT NULL,
```

Comment:
Do we need is_pedigree_ind as well? The source table declared_yield_contract_commodity actually stores its data at the Commodity-Pedigree level, so I had assumed that Verified Yield would as well?

Reply:
I'll ask Andrew

Reply:
We need to add pedigree. I'll update the documentation and release scripts